tests/TestCase/Database/FunctionsBuilderTest.php

```diff
@@ -40,7 +40,6 @@ class FunctionsBuilderTest extends TestCase
      */
     public function setUp(): void
     {
-        parent::setUp();
```

Comment:
Why was this necessary?

Reply:
Thanks, looks like I accidentally deleted that line I moved the tests out of that file.